### ISSUE TITLE
Bookmarks

### ISSUE THREAD

**Issue by Apex-DE** [Dimillian/IcySky#1]
the ability to set Bookmarks for reference and "read-later" is a thing really missing in the official client and requested many times.
Can you build it in? As a exclusive?

**Comment by Dimillian** [comment]
Indeed, I want to focus on client features for this app; this is definitely something I'm considering.
